app/templates/index.html

```diff
@@ -2,12 +2,8 @@
 
 {% block page_title %}Digital Marketplace admin{% endblock %}
 
-{% block main_content %}
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Admin</h1>
-  </div>
-</div>
+{% block content %}
+<h1 class="govuk-heading-xl">Admin</h1>
```

Comment:
Why does this heading get moved out of the div?

Reply:
There is no reason for it to be in a div (aka grid). The content is short and concise and it would display exactly the same no matter what device.